### DISCUSSION TITLE
Vertical Tabs

### DISCUSSION THREAD

**Discussion by Xelus22** [saadeghi/daisyui#2908] [Q&A]
Hi,

I saw there was a previous request for vertical tabs, and it was deemed that the `menu` be very similar. As such it wasn't implemented.
I'm having troubles combining the two as a noob.

If theres an example out there it would be greatly appreciated!

**Comment by saadeghi** [comment]
What do you mean `combining the two`? Can you explain what you want to make?

**Reply by Xelus22** [reply]
https://mantine.dev/core/tabs/#vertical-tabs-placement

Like this but with vertical tabs.

I got it working by having to accompany with some JS

**Reply by saadeghi** [reply]
We only have horizontal tabs currently. Vertical tabs will be in the roadmap for future releases.

**Reply by fballiano** [reply]
⬆ for vertical tabs 🙏

**Reply by SanmayJoshi** [reply]
+1 For vertical tabs. For an example, one can take a look at System Requirements on this page (its uses TWBS): https://softorage.com/software/adobe-photoshop/

TWBS ref: https://getbootstrap.com/docs/5.3/components/navs-tabs/#vertical

**Comment by Solido** [comment]
Is this still discussed? I've just opened a (duplicate) stating my case and this component would help a lot. To give context this the default component for every llm threads management

**Comment by pdanpdan** [comment]
I think this needs to wait for scroll markers. 
Else it's pretty cumbersome to make vertical tabs
